filter_plugins/dell_os9.py

```diff
@@ -1,19 +1,111 @@
-def os9_getLabelMap(sw_facts):
+# Method which returns a label map dictionary of Switch Label > Conf Label, or the inverse
+def os9_getLabelMap(sw_config, reverse=False):
     label_map = {}
 
-    split_conf = sw_facts.split("\n")
-    intf_conf = [i for i in split_conf if i.startswith('interface')]
+    # don't include these interface types in the label map
+    intf_blacklist = [
+        "ManagementEthernet",
+        "vlan",
+        "port-channel",
+        "group",
+        "loopback",
+        "null",
+        "tunnel"
+    ]
+
+    intf_conf = [ i for i in sw_config if i.startswith('interface') and not any(y in i for y in intf_blacklist) ]
 
     for intf in intf_conf:
         intf_parts = intf.split(" ")
-        label_map[intf_parts[2]] = " ".join(intf_parts[1:2])
+
+        sw_label = " ".join(intf_parts[1:3])
+        conf_label = intf_parts[2]
+
+        if reverse:
+            label_map[sw_label] = conf_label
+        else:
+            label_map[conf_label] = sw_label
 
     return label_map
 
-def getSpacesInStartOfString(s):
-    return len(s) - len(s.lstrip())
+# Returns a dictionary of the VLANs assigned on each interface
+def os9_getVLANAssignmentMap(vlan_dict, sw_config):
+    # helper method which takes a dell os9 interface range as an input and outputs a list as a result of the range
+    # Thanks to dell os9 this is more complicated than it has to be
```

Comment:
I would recommend the use of [docstrings](https://peps.python.org/pep-0257/) for documenting what a function/method/class does.